datasets/lt/illegal_gambling/crawler.py

```diff
@@ -0,0 +1,75 @@
+import re
+from typing import Generator, Dict, Tuple
+
+from normality import collapse_spaces
+from zavod import Context
+
+
+def parse_table(
+    table,
+    skiprows=2,
+    headers=["Number", "Name", "Domain", "Contacts", "Court information"],
+) -> Generator[Dict[str, str | Tuple[str]], None, None]:
```

Comment:
I've simplified it to not extract the URLs since that doesn't seem as useful as the full cell contents in this case - so just check if this type is still correct.